scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/InfoTreeGenerator.java

```diff
@@ -124,4 +123,8 @@ else if (ch == InfoTree.DEP_END_DELIM) {
 	 */
 	boolean canGenerate(String signature);
 
+	@Override
+	default double getPriority() {
+		return Priority.NORMAL;
```

Comment:
@gselzer Make it makes sense for the `Prioritized` interface itself to have this default?

Reply:
I kinda like the fact that it doesn't - it makes you think about what the priority should be amongst the existing implementations. And there's the fact that this was the smallest necessary change to avoid our bug.